docs/development/security.md

```diff
@@ -8,31 +8,74 @@
 
 ## Reporting Security Issues
 [//]: # (BEGIN Reporting)
-Please make sure to understand, that treating security issues confidentially is required to keep ILIAS installations as safe as possible until the issue is fixed.
+Please make sure to understand, that treating security issues confidentially is
+required to keep ILIAS installations as safe as possible until the issue is fixed.
+
+Please follow the process described in detail below. You will receive an answer
+from a member of the ILIAS security group about further steps.
 
-Please follow the process described in detail below. You will receive an answer from a member of the ILIAS security group about further steps.
 **Do not file an issue in the bugtracker!**
 
-1. Write an email to security@lists.ilias.de about your discovery, containing a description of the issue with the scenario in which the problem is triggered and a description of its implications. Please provide all necessary steps to reproduce the issue. We kindly ask you to withhold full disclosure of the issue until a fix is ready and the new release has been build and made available to everyone (full disclosure about 1 week after the new release is published).
```

Comment:
To ease communication we could use security@ilias.de instead of the longer @lists.ilias.de. Alias is already active.